plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -28,12 +35,25 @@ open class KtlintExtension {
      */
     var ignoreFailures = false
 
+    /**
+     * Report output formats.
+     *
+     * Available values: plain, plain_group_by_file, checkstyle, json.
+     *
+     * **Note** for Gradle scripts: for now all values should be uppercase due to bug in Gradle.
+     *
+     * Default is empty.
+     */
+    var reporters: Array<ReporterType> = emptyArray()
+
     /**
      * Report output format.
      *
      * Available values: plain, plain_group_by_file, checkstyle, json.
      *
-     * Default is plain.
+     * Default is none.
      */
-    var reporter: ReporterType = ReporterType.PLAIN
+    @Deprecated(message = "Ktlint introduced multi output support since 0.11.1 version",
+            replaceWith = ReplaceWith("reporters = arrayOf()", "reporter"))
+    var reporter: DeprecatedReporterType? = null
```

Comment:
Kind of a big change to change a default don't you think?

Reply:
Hm. After thinking about it again - you are right. There maybe a users that used default config and assume that `PLAIN` report will be generated.

Reply:
Tough I will leave possibility to unset the value, so plugin will not produce unneeded report type.